docs/upload-strategies/tus.mdx

```diff
@@ -0,0 +1,273 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import UppyCdnExample from '/src/components/UppyCdnExample';
+
+# Tus
+
+The `@uppy/tus` plugin brings resumable file uploading with [Tus](http://tus.io) to Uppy by wrapping the [tus-js-client][].
+
+## When should I use it?
+
+:::tip
+Not sure which upload strategy is best for you? Read “[Choosing the upload strategy you need](/docs/guides/choosing-upload-strategy)”.
+:::
+
+[Tus][tus] is an open protocol for resumable uploads built on HTTP.
+This means accidentally closing your tab or losing connection let’s you continue, for instance, your 10GB upload
+instead of starting all over.
```

Comment:
```suggestion
This means that accidentally closing your tab or losing connection lets you continue, for instance, your 10GB upload instead of starting all over.
```